UI/window-basic-about.cpp

```diff
@@ -133,8 +133,9 @@ void OBSAbout::ShowAbout()
 void OBSAbout::ShowAuthors()
 {
 	std::string path;
-	QString error = "Error! File could not be read.\n\n \
-		Go to: https://github.com/obsproject/obs-studio/blob/master/AUTHORS";
+	QString error =
+		QTStr("About.Error") + ": " +
+		"https://github.com/obsproject/obs-studio/blob/master/AUTHORS";
```

Comment:
I think a string-replacement would probably make more sense than concatenation, so something along the lines of `About.Error="[...]Go to: %1"` and then `QString error = QTStr("About.Error").arg("[link here]").